glean-core/src/metrics/labeled.rs

```diff
@@ -351,6 +353,23 @@ where
     }
 }
 
+impl<T, S> TestGetLabeledValues<S> for LabeledMetric<T>
+where
+    T: AllowLabeled + TestGetValue<S>,
+    S: Any,
+{
+    fn test_get_labeled_values(&self, ping_name: Option<String>) -> HashMap<String, S> {
+        let mut out = HashMap::new();
+        let map = self.label_map.lock().unwrap();
+        map.iter().for_each(|(label, submetric)| {
+            if let Some(v) = submetric.test_get_value(ping_name.clone()) {
+                out.insert(label.clone(), v);
+            }
+        });
+        out
+    }
+}
+
```

Comment:
I feel like there ought to be a unit test in `glean-core` somewhere. Maybe here like the metric-specific unit tests at the bottom of e.g. `counter.rs`?

Reply:
Ditto